.github/workflows/regenerate_monthly.yml

```diff
@@ -0,0 +1,36 @@
+name: Redownload & regenerate (monthly)
```

Comment:
Would turn it to a weekly though, so the changes are not huge

Reply:
With https://github.com/gtk-rs/gir-files/pull/81/commits/c856331f7f23dab4016fb100f3412e39412175ac I've since made it weekly. If this proves to be a problematic interval in the future, the cron expression on line four can always be updated later to change the frequency to whatever you wish. Thanks for your continued feedback!